mimic/test/test_server_images.py

```diff
@@ -87,9 +89,9 @@ def test_get_image_list_with_details(self):
         for each_image in image_list:
             self.assertEqual(
                 sorted(each_image.keys()),
-                sorted(['id', 'name', 'links', 'status', 'progress', 'minRam',
-                        'OS-DCF:diskConfig', 'OS-EXT-IMG-SIZE:size', 'metadata',
-                        'minDisk', 'created', 'updated']))
+                sorted(['id', 'name', 'links', 'minRam',
```

Comment:
The onMetal images test that cover this case are here: https://github.com/rackerlabs/mimic/blob/support_rackspace_images/mimic/test/test_images.py